src/app/workflows/workflows.component.ts

```diff
@@ -21,6 +21,8 @@ import { SessionService } from 'app/shared/session/session.service';
 import { Observable } from 'rxjs';
 import { EntryType } from '../shared/enum/entry-type';
 import { UrlResolverService } from '../shared/url-resolver.service';
+import { EntryTypeMetadataService } from '../entry/type-metadata/entry-type-metadata.service';
+import { EntryType as newEntryType } from 'app/shared/openapi';
```

Comment:
Suggest something like `...as OpenApiEntryType` -- camel case had me confused when I was looking at the code on line 56, since our types are generally title case, plus `new` doesn't convey a lot of info.